tests/x11/tomcat.pm

```diff
@@ -62,6 +62,120 @@ sub run() {
     my $websocket_test = Tomcat::WebSocketsTest->new();
     $websocket_test->test_all_examples();
 
+    # Close firefox
+    $self->close_firefox();
+    assert_screen('generic-desktop');
```

Comment:
Nice work Dee :) One comment from my side is that it would be nice to have a cohesion in the main function `tomcat.pm` with the previous code. What I mean here is that  you could create and call functions from `lib/Tomcat` like for example installation and configuration of apache2, test of interaction between tomcat and apache2 etc. In this way, I think it is best for the test's readability.

Reply:
Thanks Anna. I created a new file to add functionality of tomcat and apache2 via mod_jk.